plugins/meson.build

```diff
@@ -48,6 +48,7 @@ plugins = {
   'ebitdo': false,
   'elantp': false,
   'elanfp': false,
+  'elan-kbd': false,
```

Comment:
Probably not for this PR; but I think elantp and elanfp should be `elan-tp` and `elan-fp`

Reply:
Yes, I'm also thinking of renaming colorhug to hughski-colorhug too.

Reply:
sounds good